log/log.go

```diff
@@ -0,0 +1,63 @@
+package log
```

Comment:
Package needs a description...

I suspect we could put this into `runtime/log/` because it's sort of part of the runtime that is necessary to run these things...